api/HTMLElement.json

```diff
@@ -2245,10 +2245,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
The Chrome version for this (`oncopy`) is 71, which if it were true would mean this probably not in Opera 15. But it was, I tested http://mdn-bcd-collector.appspot.com/tests/api/HTMLElement/oncopy. The Chrome data is wrong.

But anyway, for all of the PRs setting the version to ≤12.1, are they things that you've confirmed to be in Opera 12.1, as well as Opera 15 and later

It would generally be useful for PR descriptions to name which browser versions were tested.